[ffmpeg] build failure using x64-windows-static

### Operating system

Windows

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
"VCPKG_TARGET_TRIPLET": "x64-windows-static",

    {
      "name": "ffmpeg",
      "features": [
        "avcodec",
        "avfilter",
        "avformat",
        "dav1d",
        "fdk-aac",
        "nvcodec",
        "openssl",
        "vpx",
        "x264",
        "x265"
      ]
    },
```

### Failure logs

CMake Error at build/vcpkg_installed/x64-windows-static/share/ffmpeg/FindFFMPEG.cmake:73 (find_library):
  Could not find FFMPEG_DEPENDENCY_crypt32_RELEASE using the following names:
  crypt32

### Additional context

_No response_

--- a/ports/ffmpeg/FindFFMPEG.cmake.in
+++ b/ports/ffmpeg/FindFFMPEG.cmake.in
@@ -45,17 +45,17 @@ function(append_dependencies out)
     cmake_parse_arguments(PARSE_ARGV 1 "arg" "DEBUG" "NAMES" "")
     if(${arg_DEBUG})
         set(config DEBUG)
-        set(path "${CURRENT_INSTALLED_DIR}/debug/lib/")
+        set(path "${SEARCH_PATH}/debug/lib/")
     else()
         set(config RELEASE)
-        set(path "${CURRENT_INSTALLED_DIR}/lib/")
+        set(path "${SEARCH_PATH}/lib/")
     endif()
     if("${CMAKE_CXX_IMPLICIT_LINK_LIBRARIES}" STREQUAL "")
         enable_language(CXX)
     endif()
     set(pass_through
         ${CMAKE_CXX_IMPLICIT_LINK_LIBRARIES}
-        bcrypt gdi32 mfuuid ole32 oleaut32 psapi secur32 shlwapi strmiids user32 uuid vfw32 ws2_32
+        advapi32 bcrypt crypt32 gdi32 mfuuid ole32 oleaut32 psapi secur32 shlwapi strmiids user32 uuid vfw32 ws2_32
         -pthread -pthreads pthread atomic m
     )
     cmake_policy(SET CMP0057 NEW)

--- a/ports/ffmpeg/vcpkg.json
+++ b/ports/ffmpeg/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "ffmpeg",
   "version": "7.1",
-  "port-version": 1,
+  "port-version": 2,
   "description": [
     "A library to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created.",
     "FFmpeg is the leading multimedia framework, able to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created. It supports the most obscure ancient formats up to the cutting edge. No matter if they were designed by some standards committee, the community or a corporation. It is also highly portable: FFmpeg compiles, runs, and passes our testing infrastructure FATE across Linux, Mac OS X, Microsoft Windows, the BSDs, Solaris, etc. under a wide variety of build environments, machine architectures, and configurations."

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2758,7 +2758,7 @@
     },
     "ffmpeg": {
       "baseline": "7.1",
-      "port-version": 1
+      "port-version": 2
     },
     "ffnvcodec": {
       "baseline": "12.2.72.0",

--- a/versions/f-/ffmpeg.json
+++ b/versions/f-/ffmpeg.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "88c48d9d8743bb6641072c4aa50e1da155c2eb50",
+      "version": "7.1",
+      "port-version": 2
+    },
+    {
       "git-tree": "8bf6bad9b522d47e259a13d9033a825a6abd41a9",
       "version": "7.1",
       "port-version": 1

